src/wallet/rpcwallet.cpp

```diff
@@ -1444,7 +1444,7 @@ UniValue listtransactions(const UniValue& params, bool fHelp)
             "    \"trusted\": xxx            (bool) Whether we consider the outputs of this unconfirmed transaction safe to spend.\n"
             "    \"blockhash\": \"hashvalue\", (string) The block hash containing the transaction. Available for 'send' and 'receive'\n"
             "                                          category of transactions.\n"
-            "    \"blockindex\": n,          (numeric) The block index containing the transaction. Available for 'send' and 'receive'\n"
+            "    \"blockindex\": n,          (numeric) The position of the transaction in the block that includes it. Available for 'send' and 'receive'\n"
```

Comment:
I'd prefer to stick with index too. It has a well-defined meaning, while position is too general (it could be a byte offset, or a set of 3D coordinates within the unit block...).